src/components/templates/CaseView/index.tsx

```diff
@@ -105,8 +105,10 @@ export default function CaseView(props) {
   // populate vertTabInfo and deferLoadInfo
   theTabsRegionChildren.forEach((tabComp, index) => {
     const theTabCompConfig = tabComp.getPConnect().getConfigProps();
-    vertTabInfo.push({name: theTabCompConfig.label, id: index});
-    deferLoadInfo.push( { type: "DeferLoad", config: theTabCompConfig } );
+    if (theTabCompConfig.visibility==true){
```

Comment:
The lint errors can be fixed by changing this line to simply:

if (theTabCompConfig.visibility) {